### ISSUE TITLE
Height and position of file/class/symbol search bar is too high

### ISSUE THREAD

**Issue by oliverswitzer** [ChrisRM/material-theme-jetbrains#1084]
### Summary

I recently installed the Material-UI plugin which adjusted the height of my search bar so that the bar itself starts at the very top of the screen. How do I revert this change? I prefer when the bar remains at the center of the screen. Check out the screenshot for details!

<img width="1917" alt="file-search-bug" src="https://user-images.githubusercontent.com/4985726/49749869-6d4ef900-fc77-11e8-819f-80ae4237da95.png">

### Expected behavior

See screenshot for the position that I *expected* the search bar to be in (which is the default, as this is where the search bar was before I added material ui plugin)

I was hoping to be able to find a configuration option for this somewhere within the Material UI config settings, but could not. 

### Environment

RubyMine 2018.3
Build #RM-183.4284.145, built on November 20, 2018
Licensed to Oliver Switzer
Subscription is active until June 25, 2019
JRE: 1.8.0_152-release-1343-b15 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.12.6
Material-UI version: v3.0.0-rc4

**Comment by mallowigi** [comment]
The plugin is not related to this change, this is the first time I’ve heard about it. Have you tried moving the search bar?

**Comment by oliverswitzer** [comment]
@mallowigi this happened the moment that I installed the plugin. It was in the correct position indicated in the picture before I installed Material-UI. 

I do not know how to move the search bar. If you know how to do this, please let me know.

**Comment by mallowigi** [comment]
From my experience you can simply drag it and it will stay there

**Comment by oliverswitzer** [comment]
@mallowigi you are correct 😮 Moving it and resizing the drop down seemed to have worked!

It is a tad strange that it changed positions and increased its height when I installed Material UI though. 

Thanks again
